### PR TITLE
Carpe Diem 2

### LINKED ISSUE #2: Fix Router
Criterios de aceptación:
- [x] Funcione Router
- [x] Que las redirecciones estén correctos

### PR DESCRIPTION
**Pull request by DuvanArwenLazar**
# Change Details 
- Migration from CLI to vite
- router works 

## Issues or Fixes 
issue #2 

## Before merging branches 

- [ ] Reviewed by one or more developers on the team.
- [ ] Tests.

### CODE REVIEW

File: una-red/src/main.js
Comment (JohnGolgota) [inline una-red/src/main.js:1]: delete

File: vite.config.js.timestamp-1705781811828-ad850e0fd6f9f.mjs
Comment (JohnGolgota) [inline vite.config.js.timestamp-1705781811828-ad850e0fd6f9f.mjs:1]: se borra

File: una-red/src/components/HelloWorld.vue
Comment (JohnGolgota) [inline una-red/src/components/HelloWorld.vue:1]: borrar

File: una-red/src/assets/vue.svg
Comment (JohnGolgota) [inline una-red/src/assets/vue.svg:1]: delite

File: una-red/src/App.vue
Comment (JohnGolgota) [inline una-red/src/App.vue:1]: fall

File: una-red/public/vite.svg
Comment (JohnGolgota) [inline una-red/public/vite.svg:1]: del

File: una-red/src/style.css
Comment (JohnGolgota) [inline una-red/src/style.css:1]: suprimir

File: una-red/.vscode/extensions.json
Comment (JohnGolgota) [inline una-red/.vscode/extensions.json:1]: del

File: src/views/Team.vue
Comment (JohnGolgota) [inline src/views/Team.vue:1]: Migrar from vue 2 syntax to vue 3 cuose vue 2 is fuckin ugly

File: README.md
Comment (JohnGolgota) [inline README.md:1]: revert this Readme

**Review by JohnGolgota — CHANGES_REQUESTED** [review]
Arregla eso

File: README.md
Comment (DuvanArwenLazar) [reply README.md:1]: complete

File: una-red/src/main.js
Comment (DuvanArwenLazar) [reply una-red/src/main.js:1]: ya

File: vite.config.js.timestamp-1705781811828-ad850e0fd6f9f.mjs
Comment (DuvanArwenLazar) [reply vite.config.js.timestamp-1705781811828-ad850e0fd6f9f.mjs:1]: ya

File: una-red/src/components/HelloWorld.vue
Comment (DuvanArwenLazar) [reply una-red/src/components/HelloWorld.vue:1]: ya

**Pull request merged**